helix-core/src/test/java/org/apache/helix/manager/zk/TestZkClusterManager.java

```diff
@@ -63,18 +63,11 @@ public void testController() throws Exception {
       _gZkClient.deleteRecursively("/" + clusterName);
     }
 
+    TestHelper.setupEmptyCluster(_gZkClient, clusterName);
+
     ZKHelixManager controller =
         new ZKHelixManager(clusterName, null, InstanceType.CONTROLLER, ZK_ADDR);
 
-    try {
-      controller.connect();
-      Assert.fail("Should throw HelixException if initial cluster structure is not setup");
```

Comment:
The problem with the previous test is that the call to get default Helix manager property will fail because we have not set it up yet. The old ZKHelixManager constructor has no dependency on cluster set up, but the new one needs to get the cloud config from it.

Reply:
I understand, but in this case, please find an alternative way to test instead of removing it.